tests/unit/VideoElement.event.spec.js

```diff
@@ -140,6 +140,43 @@ describe('VideoElement | EventHandler', function() {
 
   });
 
+it('VideoElement.onloadedmetadata :: emit', function (done) {
+    this.timeout(testItemTimeout);
+    video.id = 'video';
+
+    var now = new Date().getTime();
+
+    video.onloadedmetadata = function(event) {
+      done();
+    };
+
+    video = attachMediaStream(video, stream);
+  });
+
+  it('VideoElement.onloadedmetadata :: attributes', function (done) {
+    this.timeout(testItemTimeout);
+    video.id = 'video';
```

Comment:
Can you make the id a random string ?
I'm afraid some day we might end up with false positive in the future.

Math.random().toString(36).slice(2);